### DISCUSSION TITLE
Goerli test network connection

### DISCUSSION THREAD

**Discussion by ahmadkhan345** [wevm/wagmi#1981] [General]
How can I connect to the goerli test network using its chaidid?
  const { connect } = useConnect({
    connector: new MetaMaskConnector(),
    chainId: 5,
  })
This is how I was trying it.

**Comment by challet** [comment]
I think you got it right, at least this `chainId` exists as one of the useConnect option : https://wagmi.sh/react/hooks/useConnect#chainid-optional

**Comment by solvman** [comment]
Goerli chain already exists in "wagmi/chains", there is no need to hard code that '5' in there. You can do:
```javascript
import { goerli } from "wagmi/chains";
import { useConnect } from "wagmi";
import { MetaMaskConnector } from "wagmi/connectors/metaMask";

const { connect } = useConnect({connector: new MetaMaskConnector({chains:goerli.id}), chainId: goerli.id});

```
